ogc/tiles/templates/tileMatrixSets/EuropeanETRS89_LAEAQuad.go.html

```diff
@@ -11,7 +11,7 @@ <h1 class="title">EuropeanETRS89_LAEAQuad</h1>
     <table class="table table-striped">
       <thead>
         <tr>
-          <th>Zoomniveau (tile matrix id)</th>
+          <th>Zoomniveau<br/>(tile matrix id)</th>
```

Comment:
Beetje muggenzifterij, maar zouden we Zoomniveau hier niet in de i18n moeten zetten?

Reply:
Goed gezien, geen muggenzifterij wat mij betreft. Heb het gefixed.